EIPS/eip-BN256_HashToCurve.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: TBD
+title: Precompile for BN256 HashToCurve Algorithms
+author: Dr. Christopher Gorman, MadHive (@chgormanMH)
```

Comment:
```suggestion
author: Dr. Christopher Gorman MadHive (@chgormanMH)
```
This is likely going to break something, as the `,` character is special in the author line.  I'm not sure what `MadHive` is, but is this suggestion reasonable?

Reply:
I have removed `, MadHive`. I work at MadHive. Is there a standard practice of including one's employer? I developed this work during my employment.

Reply:
EIPs are all CC0, so there is no attribution.  The author line is mainly just for access control on who can freely make changes to the EIP leading up to final.  The author line is left in for final EIPs, but it is pretty meaningless given the license.

Reply:
Not a problem, then.